orangecontrib/text/datasets/url.tab

```diff
@@ -0,0 +1,19 @@
+topic	url_case_study	url

```

Comment:
each in it's own row ?
the first row should contain attribute names right?
 url. topic and url case study are attribute names for the corresponding columns that contain urls